CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Enhancements
+
+- Set `http.response.status_code` to `span.data` ([#1497](https://github.com/getsentry/sentry-dart/pull/1497))
```

Comment:
We can be more generic here now since there are more fields, or list all of them